Assets/Scripts/AudioManager.cs

```diff
@@ -130,7 +130,7 @@ private void RegisterEvents()
                 AudioSource.PlayClipAtPoint(mPlayerDeath, pos);
             };
 
-            EventSystem.OnJumpEvent += () =>
+            EventSystem.OnJumpEvent += (Vector3 pos) =>

```

Comment:
The jump event only fires for the local player so in the audio manager we don't need to do 3D sound or anything therefore we leave it unused. We use that pos parameter in the DustCloud script though which is why all listeners need it